relay-dynamic-config/src/feature.rs

```diff
@@ -10,6 +10,11 @@ pub enum Feature {
     /// Serialized as `organizations:session-replay`.
     #[serde(rename = "organizations:session-replay")]
     SessionReplay,
+    /// Enables ingestion of mobile Session Replays.
+    ///
+    /// Serialized as `organizations:session-replay-mobile`.
+    #[serde(rename = "organizations:session-replay-mobile")]
+    SessionReplayMobile,
```

Comment:
Is it `replay.organizations.mobile-slug-denylist` (from the PR description) or `organizations:session-replay-mobile`? I couldn't find any matches in (get)sentry.

Reply:
Renamed to `organizations:session-replay-video`.